LayoutTests/imported/w3c/web-platform-tests/resources/testdriver-vendor.js

```diff
@@ -289,45 +285,46 @@ window.test_driver_internal.action_sequence = function(sources)
     logDebug(() => JSON.stringify(pointerSource));
 
     if (pointerType === "touch")
-        return dispatchTouchActions(pointerSource.actions);
+        await dispatchTouchActions(pointerSource.actions);
     if (testRunner.isIOSFamily && "createTouch" in document)
-        return dispatchTouchActions(pointerSource.actions, { insertPauseAfterPointerUp: true });
-    if (pointerType === "mouse" || pointerType === "pen")
-        return dispatchMouseActions(pointerSource.actions, pointerType);
+        await dispatchTouchActions(pointerSource.actions, { insertPauseAfterPointerUp: true });
+    if (["mouse", "pen"].includes(pointerType))
+        await dispatchMouseActions(pointerSource.actions, pointerType);
```

Comment:
This is a behavior difference fwiw, `return` stops the function execution, and prevents from going to the next `if` branch. `await` doesn't.

Reply:
Oh you are right. Oops.